lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/CompletedTaskBridge.java

```diff
@@ -22,16 +22,14 @@
 
 public class CompletedTaskBridge extends NativeBridge {
 
-  private CompletedTask jcompletedTask;
+  private final CompletedTask jcompletedTask;
 
-  private String taskId;
+  private final String taskId;
 
-  private ActiveContextBridge jactiveContext;
 
-  public CompletedTaskBridge(CompletedTask completedTask) {
+  public CompletedTaskBridge(final CompletedTask completedTask) {
     jcompletedTask = completedTask;
     taskId = completedTask.getId();
-    jactiveContext = new ActiveContextBridge(completedTask.getActiveContext());
```

Comment:
why remove this completely? shouldn't this be replaced by the activeContextBridgeFactory.Get(completedTask.getActiveContext().getId())?